Binary files orig/png/src/.Makevars.win.swp and patched/png/src/.Makevars.win.swp differ
diff -Nru orig/png/src/Makevars.win patched/png/src/Makevars.win
--- orig/png/src/Makevars.win	2022-11-29 10:42:31.000000000 +0100
+++ patched/png/src/Makevars.win	2023-10-10 09:30:40.252708900 +0200
@@ -1,7 +1,13 @@
 ## detect 64-bit Windows
 ifeq ($(strip $(shell $(R_HOME)/bin/R --slave -e 'cat(.Machine$$sizeof.pointer)')),8)
 PKG_CPPFLAGS=-Iwin64
-PKG_LIBS=-Lwin64 -lpng -lz
+
+ifeq (,$(shell pkg-config --version 2>/dev/null))
+  PKG_LIBS=-Lwin64 -lpng -lz
+else
+  PKG_LIBS=$(shell pkg-config --libs libpng)
+endif
+
 else
 PKG_CPPFLAGS=-Iwin32
 PKG_LIBS=-Lwin32 -lpng -lz
